app/src/main/java/org/mozilla/focus/utils/GifImageView.java

```diff
@@ -0,0 +1,92 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Source taken from https://stackoverflow.com/questions/42898968/adding-gif-in-android-studio */
```

Comment:
nit: Sorry if I was unclear - this should not be in the license header – it should be in the class Javadoc:
```java
/*
 * License
 */

package ...

import ...

/**
 * Source taken from...
 */
public class GifImageView ... {
    ...
}
```